src/test/java/io/vertx/core/http/HttpBandwidthLimitingTest.java

```diff
@@ -204,6 +205,54 @@ public void start(Promise<Void> startPromise) {
     Assert.assertTrue(elapsedMillis > expectedTimeMillis(totalReceivedLength.get(), OUTBOUND_LIMIT)); // because there are simultaneous 2 requests
   }
 
+  @Test
+  public void testDynamicOutboundRateUpdateSharedServers() throws IOException, InterruptedException
+  {
+    int numEventLoops = 5; // We start a shared TCP server with 2 event-loops
+    List<HttpServer> servers = new ArrayList<>();
```

Comment:
the list should be synchronised because it will be updated concurrently by different threads.